pkgs/applications/audio/cava/default.nix

```diff
@@ -1,26 +1,46 @@
-{ stdenv, fetchgit, alsaLib, fftw }:
+{ stdenv, fetchFromGitHub, autoconf, automake, libtool, alsaLib, fftw,
+  libpulseaudio, ncurses }:
 
 stdenv.mkDerivation rec {
   name = "cava-${version}";
-  version = "27dbdf47daae44c780db9998c760007b3bf63738";
+  version = "0.4.1";
 
-  buildInputs = [ alsaLib fftw ];
+  buildInputs = [
+    autoconf
+    automake
+    libtool
+    alsaLib
+    fftw
+    libpulseaudio
+    ncurses
+  ];
 
-  src = fetchgit {
-    url = "https://github.com/karlstav/cava";
+  src = fetchFromGitHub {
+    owner = "karlstav";
+    repo = "cava";
     rev = version;
-    sha256 = "1a61e2c869376276cf78e6446cd1cc7f96b3e378fa8bc0bc4c5ca81945429909";
+    sha256 = "157hw4cn3qjic7ymn5vy67paxmzssc33h1zswx72ss7j6nc8707f";
   };
 
-  installPhase = ''
-    mkdir -p $out/bin
-    cp cava $out/bin
+  preConfigure = ''
+    # Do stuff that autogen.sh would normally do. autogen.sh tries to copy
+    # the example config into the home directory, so we can't run it directly.
+    libtoolize
+    aclocal
+    autoconf
+    automake --add-missing
+  '';
+
+  postConfigure = ''
+    substituteInPlace Makefile \
+      --replace "-L/usr/local/lib -Wl,-rpath /usr/local/lib" ""
   '';
 
   meta = with stdenv.lib; {
     description = "Console-based Audio Visualizer for Alsa";
     homepage = https://github.com/karlstav/cava;
-    maintainers = with maintainers; [offline];
+    license = licenses.mit;
+    maintainers = with maintainers; [ mirrexagon ];
```

Comment:
Do you know if `offline` wants to drop maintainership ? Otherwise I would suggest keeping both.